framework/src/main/java/org/checkerframework/framework/util/FlowExpressionParseUtil.java

```diff
@@ -158,21 +153,102 @@ private static Pattern anchored(String regex) {
     }
 
     private static boolean isMemberSelect(String s, FlowExpressionContext context) {
-        return parseMemberSelect(s) != null;
+        return parseMemberSelect(s, context) != null;
+    }
+
+    /**
+     * Matches field accesses whose receiver is a method call expression. First of returned pair is
+     * a method call expression and the second is a field or method call expression.
+     *
+     * @param sWithFormalParamNames expression string with replaced occurences of one-based
+     *     parameter index of with formal parameter names
+     * @param s expression string
+     * @param argumentList list of formal parameter names corresponding to index of formal
+     *     parameters in the method call that is in s
+     * @return pair of method call expression and field, or two method call expressions
```

Comment:
The method body returns `null` sometimes, but the documentation does not indicate that is possible nor state when it occurs.